app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -93,16 +93,19 @@ def ensure_managers_zone_and_provider_region
     if network_manager
```

Comment:
I don't like the name `ensure_managers_zone_and_provider_region` since as we keep more attributes in sync the method will become comically long.  `sync_child_managers` or something?

Reply:
Could we move those callbacks into the individual managers somehow?

Reply:
I don't think we can have a callback on the NetworkManager fire when the CloudManager changes